relay-metrics/src/lib.rs

```diff
@@ -112,3 +112,4 @@ mod statsd;
 
 pub use aggregation::*;
 pub use protocol::*;
+pub use statsd::*;
```

Comment:
Please revert this, this module is supposed to be strictly internal like in all other crates. Either ensure that such metrics can be logged internally, or define them in a different place.